cmd/utils/flags.go

```diff
@@ -209,6 +212,31 @@ var (
 		Usage: "Dashboard metrics collection refresh rate",
 		Value: dashboard.DefaultConfig.Refresh,
 	}
+	// Stateth settings
+	StatethDockerPrefixFlag = cli.StringFlag{
+		Name:  "stateth.dockerprefix",
+		Usage: "Prefix to be used for docker network and containers. Must be unique.",
+		Value: stateth.DefaultConfig.DockerPrefix,
+	}
+	StatethDashboardsFolderFlag = cli.StringFlag{
+		Name:  "stateth.dashboardsfolder",
+		Usage: "Grafana dashboards folder",
+		Value: stateth.DefaultConfig.DashboardsFolder,
+	}
+	StatethGrafanaPortFlag = cli.IntFlag{
+		Name:  "stateth.grafanaport",
+		Usage: "Grafana http port",
+		Value: stateth.DefaultConfig.GrafanaPort,
+	}
+	StatethInfluxdbPortFlag = cli.IntFlag{
+		Name:  "stateth.influxdbport",
+		Usage: "Influxdb http port",
+		Value: stateth.DefaultConfig.InfluxDBPort,
+	}
+	StatethRmFlag = cli.BoolFlag{
+		Name:  "stateth.rm",
+		Usage: "Remove existing stateth network and stateth containers upon startup. Make sure that the start up works every time, even if the service wasn't shut down gracefully.",
```

Comment:
As suggested in another comment, maybe we should have a default value `true` here. What do you think?